frontend/public/locales/en/public.json

```diff
@@ -728,5 +728,11 @@
   "OK": "OK",
   "Last updated": "Last updated",
   "Disabled": "Disabled",
-  "Enabled": "Enabled"
-}
\ No newline at end of file
+  "Enabled": "Enabled",
+  "LogInToYourAccount": "Log in to your account",
+ 	"LogIn": "Log in",
+ 	"LoginTitle": "Login",
+ 	"WelcomeTo": "Welcome to {{platformTitle}}",
+ 	"LogInWith": "Log in with {{providerName}}",
```

Comment:
@rebeccaalpert I included `{{platformTitle}}` and `{{providerName}}` to give the translators context. Is this acceptable?

Reply:
Works for me! Thanks.

Reply:
It's better if the phrase is the key on the left. The translators only receive the key value and they're not going to be able to translate LoginTitle, etc.

Reply:
updated the keys
```
"Log in to your account": "Log in to your account",
"Log in": "Log in",
"Login": "Login",
"Welcome to {{platformTitle}}": "Welcome to {{platformTitle}}",
"Log in with {{providerName}}": "Log in with {{providerName}}",
"Error": "Error"
```

Reply:
Thank you!